examples/amp-story-360.amp.html

```diff
@@ -17,20 +17,27 @@
         publisher="AMP Story"
         publisher-logo-src="example.com/logo.png"
         poster-portrait-src="example.com/poster.jpg">
-      <amp-story-page id="cover">
+      <amp-story-page id="page1">
         <amp-story-grid-layer template="fill">
-          <amp-story-360 layout="fill" heading-start="-45" pitch-start="-20" heading-end="95" pitch-end="-10" zoom-end="4" duration="30s">
-            <amp-img src="img/panorama1.jpg" width="7168" height="3584"></amp-img>
+          <amp-story-360 layout="fill" heading-start="45" pitch-start="-20" heading-end="115" pitch-end="10" zoom-end="1.5" duration="3s" controls="gyroscope">
+            <amp-img src="img/SeanDoran-Quela-sol1462-edited_ver2-sm.png" width="1998" height="999"></amp-img>
```

Comment:
Panorama1 I'm not sure. I'll remove it until we have clarification.

`SeanDoran-Quela-sol1462-edited_ver2-sm.png` is an image from NASA.
Since we're using it for educational/informational purposes, I think it's okay:
https://www.nasa.gov/multimedia/guidelines/index.html